package.js

```diff
@@ -0,0 +1,22 @@
+// package metadata file for Meteor.js
+
+Package.describe({
+  name: 'twbs:bootstrap',  // http://atmospherejs.com/twbs/bootstrap
+  summary: 'The most popular front-end framework for developing responsive, mobile first projects on the web.',
+  version: '3.3.2',
+  git: 'https://github.com/twbs/bootstrap.git'
+});
+
+Package.onUse(function (api) {
+  api.versionsFrom('METEOR@1.0');
+	api.use('jquery', 'client');
```

Comment:
This line asks Meteor to let this package see the `jquery` package (a core Meteor package actually) which exports the jQuery object [you need in your `dist/js/bootstrap.js`](https://github.com/twbs/bootstrap/blob/master/dist/js/bootstrap.js#L7).

...I've just updated it to better specify that the requirement is only for client code.